### ISSUE TITLE
If multiple maps on a page all polycolor lines render to only one of the maps

### ISSUE THREAD

**Issue by alfa07** [Oliv/leaflet-polycolor#1]
This leads to shared renderer between polycolor instances if it is not specified explicitly:
```
    options: {
      colors: [],
      renderer: new Renderer()
    },
```
As a workaround providing renderer at polycolor call site
```
L.polycolor(
    points.map(trkpt => [trkpt.latitude,trkpt.longitude]), 
    {
      colors: colorGradient,
      weight: 5,
      renderer: new L.Renderer.RendererGradient()
    }
  ).addTo(map);
```
fixes the issue

**Comment by KirillMetrik** [comment]
+1

is there any plan to fix this?

**Comment by KirillMetrik** [comment]
@Oliv  The following modification of Polycolor object should fix the issue:
```
    options: {
      colors: []
    },

    initialize: function(latlngs, options) {
      this.options.renderer = new Renderer();
      L.Util.setOptions(this, options);
      this._setLatLngs(latlngs);
      this._colorParts = [];
    }
```
Note the `this.options.renderer = new Renderer();` line.

**Comment by Oliv** [comment]
@alfa07 @KirillMetrik Hello there, this is fixed in [0.9.1](https://github.com/Oliv/leaflet-polycolor/releases/tag/0.9.1). 
Thanks for the report !

**Comment by KirillMetrik** [comment]
@Oliv it seems like creating a separate renderer for each polyline is not correct. When I have a lot of polylines on map Chrome degrades dramatically with endless rerendering. Going back to shared renderer fixes that. So I am not quite sure how to fix this correctly but the fix I suggested is probably not a good idea.

**Comment by Oliv** [comment]
@KirillMetrik OK, I'm going to do some further tests

**Comment by KirillMetrik** [comment]
Probably the best would be to do `L.SVG.include` instead of completely custom renderer but in this case the whole rendering will need to be changed to SVG instead of Canvas.

**Comment by Oliv** [comment]
This has been fixed in 2.0.0, I use the default Map renderer now.

Only the Canvas renderer is supported right now, I plan to add SVG if needed.

Regards
